app/services/store.js

```diff
@@ -60,4 +61,17 @@ export default class ExtendedStoreService extends Store {
       'filter[:uri:]': uri,
     });
   }
+
+  // TODO: Move this to a concept-service where we bundle logic for getting
+  // all concepts of a scheme/specific concepts? Might cut down some of the
+  // repetition across the codebase.
+  queryConceptsForConceptScheme(conceptSchemeUri) {
+    return this.query('concept', {
+      'filter[concept-schemes][:uri:]': conceptSchemeUri,
+      'filter[:has-no:narrower]': true,
+      include: 'broader,narrower',
+      'page[size]': PAGE_SIZE.CODE_LISTS,
+      sort: 'position',
+    });
+  }
```

Comment:
This query for concepts is quite specific (this exact query was only used for `VERGADERACTIVITEIT` before).
For generic use, the filter on no `narrower`, the `includes` and `sort` should be dynamic.
```
'filter[:has-no:narrower]': true,
include: 'broader,narrower',
sort: 'position',
```
Right now, the only reason to even sort on `-label` is because of `AGENDA_ITEM_TYPES`. 
If we add `position` there as well, problem sorted? `Nota` first, `Mededeling` second

If we never have to reverse sort on `position` then you can just `sort: 'position',` by default

Reply:
of course, whatever we do in frontend has to match against the cache from the service.

Reply:
I'm ok with adding this, just a bit concerned that it's going to be annoying down the line. Having the query set in stone behind a method means that there's no way to accidentally miss the cache. If we make it customizable, we risk missing the cache if changes happen in the frontend, but not in the cache-warmup-service.